libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerOptions.cs

```diff
@@ -87,8 +87,5 @@ public QnAMakerOptions()
         /// <seealso cref="RankerTypes"/>
         [JsonProperty("rankerType")]
         public string RankerType { get; set; }
-
-        [JsonProperty("metadataBoost")]
-        public Metadata[] MetadataBoost { get; set; }
```

Comment:
You can't do this. This is a breaking change.  What you need to do is to add attribute

[Obsolete("This field is no longer used")]
[JsonIgnore]
 public Metadata[] MetadataBoost { get; set; }

And update add comment describing that it is not longer used.